HTTP_FileServer.js

```diff
@@ -1,11 +1,11 @@
 
-var fs = require('fs');
-var http = require('http')
-var portNumber = process.argv[2];
+const fs = require('fs');
```

Comment:
if you aren't going to ever change the variable you should use const instead of var.